job_spec/WATER_MAP.yml

```
@@ -132,7 +156,6 @@ WATER_MAP:
         - Ref::iterative_min
         - --iterative-max
         - Ref::iterative_max
-        - Ref::granules
-      timeout: 36000
+      timeout: TODO
       vcpu: 1
       memory: 31600
```

```suggestion
      vcpu: 1
      memory: 126000
```
looks like we upped the memory for water map jobs int he river width spec, so I suspect asf-tools needs a lot of memory for 10m water maps.